esp-hal/src/soc/psram_common.rs

```diff
@@ -23,3 +25,16 @@ impl PsramSize {
         matches!(self, PsramSize::AutoDetect)
     }
 }
+
+/// Returns the address range available in external memory.
+#[cfg(any(feature = "quad-psram", feature = "octal-psram"))]
+pub fn psram_range(_psram: &crate::peripherals::PSRAM) -> Range<usize> {
+    unsafe { super::MAPPED_PSRAM.memory_range.clone() }
+}
+
+/// Returns the address range available in external memory.
+#[cfg(any(feature = "quad-psram", feature = "octal-psram"))]
+pub fn psram_pointer(psram: &crate::peripherals::PSRAM) -> (*mut u8, usize) {
```

Comment:
not sure about the naming here

Reply:
Me neither, I could use some help with the name :)

Reply:
Maybe psram_raw?